packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/translations/Messages_fr_CA.json

```diff
@@ -1,6 +1,6 @@
 {
-  "telephony_alias_ovh_pabx_tts_breadcrumb": "Gestion des TTS",
-  "telephony_alias_ovh_pabx_tts_title": "Gestion des TTS (Text To Speech)",
+  "telephony_alias_ovh_pabx_tts_breadcrumb": "Gérer des TTS",
+  "telephony_alias_ovh_pabx_tts_title": "Gérer des TTS (Text To Speech)",
```

Comment:
Especially for this translation, shouldn't be done into the `Messages_fr_FR.json` instead?